recipes/graphormer_pretrained/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "graphormer-pretrained" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/graphormer-pretrained-{{ version }}.tar.gz
+  sha256: 301b7916216995b581e5599bcd3dbbdfe668efd08f35f8c7a9cac64a2681d191
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools
+    - wheel
+    - cython
+    - numpy
+    - pip
+  run:
+    - python
+    - tqdm
+    - gcsfs
+    - pytorch >=1.10.2
```

Comment:
this is going to give troubles on windows because we don't have a package for pytorch on conda-forge.

Maybe add this to a run_constraint? that way, you would be able to download this package from conda-forge, and pytorch from the pytroch channel on windows.

Otherwise, please skip the windows builds.

Reply:
I am going to skip windows for now. The CI is still failing since we need to wait for the latest fairseq builds to be available.